docs/topics/con_introduction-to-productname.adoc

```diff
@@ -8,10 +8,10 @@ This section provides a brief overview of the foundational concepts behind {Prod
 * A collection of runtimes, such as {WildFlySwarm} and {SpringBoot}, designed to run on OpenShift.
 * A prescriptive approach to cloud-native development on OpenShift.
 
-The goal of {ProductName} is to provide opinionated architectures, design patterns, tools, and best practices in ready-made example applications you can execute on OpenShift to enable cloud-native development.
+The goal of {ProductName} is to provide prescriptive architectures, design patterns, tools, and best practices in ready-made example applications you can execute on OpenShift to enable cloud-native development.
 
 .Cloud-native Development
-Cloud-native development is an approach to developing, deploying, and running applications that fully embrace the advantages of cloud computing.
+Cloud-native development is an approach to developing, deploying, and running applications that fully take advantage of cloud computing.
```

Comment:
"fully take advantage" -- would it be better to say "take full advantage"?

Reply:
I suppose it is. Thanks.